docs/scaling/multi-node-setup.rst

```diff
@@ -4,162 +4,237 @@
 CrateDB multi-node setup
 ========================
 
-CrateDB is a distributed datastore and so in production, a cluster will
-typically consist of 3 or more nodes. CrateDB makes cluster setup as easy as
-possible, but there are things to note when building a new cluster.
+CrateDB can run on a single node. However, in most environments, CrateDB is run
+as a cluster of three or more nodes.
 
-CrateDB is designed in a shared nothing architecture, in which all nodes are
-equal and each node is self-sufficient. This means that nodes work on their
-own, and all nodes in a cluster are configured equally, the same as with a
-single-node instance.
+For development purposes, CrateDB can :ref:`auto-bootstrap
+<auto-bootstrapping>` a cluster when you run all nodes on the same host.
+Auto-bootstrapping requires zero configuration. However, in production
+environments, you must the bootstrapping process :ref:`manually
```

Comment:
> you must the bootstrapping process

Is there a word missing here? You must configure the bootstrapping process, or something like that?